lib/cdo/aws/cloudfront.rb

```diff
@@ -151,7 +151,13 @@ def self.distribution_config(app, origin, aliases, ssl_cert=nil)
               ErrorCode: error,
               ResponseCode: error,
               ResponsePagePath: '/assets/error-pages/site-down.html'
-            }
+            }.tap do |error_response_hash|
+              if rack_env?(:levelbuilder)
```

Comment:
Can we leverage our new config system rather than using the environment directly here?

On code style - I'm not sure I like this use of `tap` and on deleting entries from the hash, but I could be convinced.

Reply:
Good point.  I've switched to checking the hostname.

I tried some alternatives before switching to using `tap`, which seems to be a common solution for conditionally building a literal hash in Ruby.  One approach I tried initially
 was conditionally setting each key, but that resulted in repeated verbose conditionals:
``` ruby
              ErrorCachingMinTTL: (CDO.pegasus_hostname == 'levelbuilder.code.org', 0, ERROR_CACHE_TTL),
              ErrorCode: error,
              ResponseCode: (CDO.pegasus_hostname == 'levelbuilder.code.org', nil, error),
              ResponsePagePath: (CDO.pegasus_hostname == 'levelbuilder.code.org', nil, /assets/error-pages/site-down.html')
```

Reply:
If only I was generating YAML in an ERB! :)